### PR TITLE
Web client integration

### PR DESCRIPTION
**Pull request by Jahnavi-pabbathi**

### CODE REVIEW

**Review by anushka1403 — APPROVED** [review]

**Review by anushka1403 — APPROVED** [review]

**Review by ISHAASTHANA — APPROVED** [review]

**Review by Pavan-KalyanT — APPROVED** [review]
Tested with this .yml file. No issues

**Review by pavitra001 — COMMENTED** [review]
reviewed

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@RestController
+@RequestMapping("library_service_api/v1")
+public class LibrarySystemController {
+    private final LibraryService libraryService;
+    public LibrarySystemController(LibraryService libraryService){
+        this.libraryService=libraryService;
+    }
+
+    @GetMapping("getAllBooks")
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:20]: No verbs in API name.

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    public List<Book> getAllBooks(){
+        return libraryService.getAllBooks();
+    }
+
+    @PostMapping("addBook")
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:25]: No verbs in API name.

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java (hunk 1 above)
Comment (ananya-anni) [reply LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:20]: Done

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java (hunk 2 above)
Comment (ananya-anni) [reply LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:25]: Sure

**Pull request merged**